library/command/client_command/src/main/java/org/quiltmc/qsl/command/impl/client/ClientCommandInternals.java

```diff
@@ -90,7 +97,7 @@ public static boolean executeCommand(String message) {
 
 		// The interface is implemented on ClientCommandSource with a mixin.
 		// noinspection ConstantConditions
-		QuiltClientCommandSource commandSource = (QuiltClientCommandSource) client.getNetworkHandler().getCommandSource();
+		var commandSource = (QuiltClientCommandSource) client.getNetworkHandler().getCommandSource();
```

Comment:
Not pertinent but what's the stance on interface injection in QSL, to eliminate things like this? I saw that it was used once in some other module and then removed after there were issues, are we avoiding it for now?

Reply:
Those issues were fixed, so we can definitely use injected interfaces now